python/ray/tests/test_plasma_unlimited.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
+import os
+
+import ray
+from ray.test_utils import wait_for_condition
+from ray.internal.internal_api import memory_summary
+
+MB = 1024 * 1024
+
+
+def _init_ray():
+    return ray.init(
+        num_cpus=2,
+        object_store_memory=700e6,
+        _system_config={"plasma_unlimited": 1})
+
+
+def _check_spilled_mb(address, spilled=None, restored=None):
+    def ok():
+        s = memory_summary(address=address["redis_address"], stats_only=True)
+        print(s)
+        if restored:
+            if "Restored {} MiB".format(restored) not in s:
+                return False
+        else:
+            if "Restored" in s:
+                return False
+        if spilled:
+            if "Spilled {} MiB".format(spilled) not in s:
+                return False
+        else:
+            if "Spilled" in s:
+                return False
+        return True
+
+    wait_for_condition(ok, timeout=3, retry_interval_ms=1000)
+
+
+def test_fallback_when_spilling_impossible_on_put():
+    try:
+        address = _init_ray()
```

Comment:
Can you just make `init_ray` a fixture?

Reply:
Won't do--- this is easy enough to understand (pytest fixtures are awful).